restli-int-test/src/test/java/com/linkedin/restli/examples/TestGreetingUnstructuredDataResources.java

```diff
@@ -114,4 +112,32 @@ public void testGetInternalServiceException(String resourceURL)
       assertEquals(conn.getResponseCode(), 500);
     });
   }
+
+  @DataProvider(name = "exceptionNoContentURLs")
+  private static Object[][] exceptionNoContentURLs()
+  {
+    return new Object[][] {
+        { "/greetingAssociationUnstructuredData/src=exception_204&dest=bar" }
+    };
+  }
+
+  @Test(dataProvider = "exceptionNoContentURLs")
+  public void testGetInternalServiceExceptionWithNoContent(String resourceURL)
+      throws Throwable
+  {
+    sendGet(resourceURL, (conn) -> {
```

Comment:
nit: format: { should be in the following line
same as following.

Reply:
I think the lambda in pegasus is doing this way